packages/eslint-config-godaddy-react-typescript/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "eslint-config-godaddy-react-typescript",
+  "version": "1.0.0",
+  "description": "ESLint config for consistent style in ES6 React projects using TypeScript at GoDaddy.",
+  "scripts": {
+    "lint": "eslint .",
+    "pretest": "npm run --silent lint",
+    "test": "echo ok",
+    "unused": "eslint-find-rules --unused ./index.js || echo ''"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:godaddy/javascript.git"
+  },
+  "keywords": [
+    "godaddy",
+    "javascript",
+    "styleguide",
+    "style-guide",
+    "eslint",
+    "es6",
+    "react",
+    "typescript"
+  ],
+  "bin": {
+    "eslint-godaddy-react-typescript": "bin/eslint-godaddy-react-typescript"
+  },
+  "dependencies": {
+    "eslint-config-godaddy-react": "^5.0.0"
+  },
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.3.1",
```

Comment:
I would avoid this anti-pattern, which has now been fixed in https://github.com/godaddy/javascript/pull/89 for the existing code. Let's have eslint plugins as regular dependencies, not force consuming apps to install these plugins. The only peer dependency should be eslint and probably typescript in our case.

Reply:
Take a look at the latest version of this and see if it aligns with what you have in mind.